floors/floors.go

```diff
@@ -68,34 +67,37 @@ func updateBidRequestWithFloors(extFloorRules *openrtb_ext.PriceFloorRules, requ
 		return []error{}
 	}
 
+	if err := validateSchemaDimensions(modelGroup.Schema.Fields); err != nil {
+		return []error{err}
+	}
+
```

Comment:
Do you think this belongs in `selectValidFloorModelGroups` since that is where we are validating the model groups?

Reply:
yes it can be validated in `selectValidFloorModelGroups`